lawnchair/res/values/strings.xml

```diff
@@ -231,6 +231,8 @@
 
     <string name="force_rounded_widgets">Rounded Corners</string>
     <string name="allow_widget_overlap">Allow Overlap</string>
+    <string name="force_widget_resize">Make all widgets resizable</string>
+    <string name="widget_unlimited_size">Unlimited widget size</string>
```

Comment:
```suggestion
    <string name="force_widget_resize">Enforce Widget Resizing</string>
    <string name="widget_unlimited_size">Remove Size Constraints</string>
```

Lawnchair uses Title Case for most of its UI text. However, I'd suggest tweaking the text a bit more.

I'd also suggest adding a short description:
```
Enforce Widget Resizing
Allow resizing of widgets that are constrained to a specific size

Remove Size Constraints
Remove the minimum and maximum size restrictions of widgets
```

Reply:
I'll implement this (hopefully later today).